defi/ui-tool/src/App.js

```
@@ -321,12 +356,21 @@ const App = () => {
+                  maxHeight: '600px',
+                  overflow: 'auto',
+                  fontFamily: 'monospace',
+                  fontSize: 12,
+                  whiteSpace: 'pre-wrap',
+                }}
                 ref={outputRef}
-                className="output-container"
-              >
-                <pre>{output || ""}</pre>
-              </div>
+                dangerouslySetInnerHTML={{ __html: parseAnsiToHtml(output || '') }}
```

Biome (2.3.13)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)